pkg/status/status.go

```diff
@@ -174,7 +194,7 @@ func (m *Prober) IsReady(ctx context.Context, ing *v1alpha1.Ingress) (bool, erro
 		m.mu.Lock()
 		defer m.mu.Unlock()
 		if state, ok := m.ingressStates[ingressKey]; ok {
-			if state.hash == hash {
+			if state.version == hash {
```

Comment:
Rename the hash on the right as well to make it consistent?

Reply:
Done in a later commit